doc/ref_arch/openstack/chapters/chapter01.md

```diff
@@ -1,65 +1,51 @@
-[<< Back](../../openstack)
 
-# 1. Introduction
+# Introduction
 
-## Table of Contents
-* [1.1 Overview](#1.1)
-  * [1.1.1 Vision](#1.1.1)
-* [1.2 Use Cases](#1.2)
-* [1.3 Anuket OpenStack Reference Release](#1.3)
-* [1.4 Principles](#1.4)
-  * [1.4.1	OpenStack specific principles](#1.4.1)
-* [1.5 Document Organisation](#1.5)
-* [1.6 Terminology](#1.6)
+## Overview
 
-
-<a name="1.1"></a>
-## 1.1 Overview
 This Reference Architecture is focussed on OpenStack as the Virtualised Infrastructure Manager (VIM) chosen based on the criteria laid out in the [Reference Model](../../../ref_model/chapters/chapter01.md). [OpenStack](https://docs.openstack.org) has the advantage of being a mature and widely accepted open-source technology; a strong ecosystem of vendors that support it, the OpenInfra Foundation for managing the community, and, most importantly, it is widely deployed by the global operator community for both internal infrastructure and external facing products and services.  This means that the operators have existing staff with the right skill sets to support a Cloud Infrastructure ([NFVI](https://www.etsi.org/deliver/etsi_gs/NFV-INF/001_099/001/01.01.01_60/gs_NFV-INF001v010101p.pdf)) deployment into development, test and production. Another reason to choose OpenStack is that it has a large active community of vendors and operators, which means that any code or component changes needed to support the Common Telco Cloud Infrastructure requirements can be managed through the existing project communities' processes to add and validate the required features through well-established mechanisms.
 
-<a name="1.1.1"></a>
-## 1.1.1. Vision
+## Vision
```

Comment:
Wasn't this supposed to be ### Vision?

Reply:
@ildikov Yes, thanks, good catch! I have modified